### PR TITLE
feat: added grids

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

File: css/grid-3.css [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  border: 2px solid var(--dark-purple);
+}
+
+.cart-manage-container .change {
+  border: 2px solid var(--black);
+  padding: 5px;
+  height: max-content;
+  width: max-content;
+  font-size: 1.5rem;
+  font-weight: normal;
```
Comment (sumitk-singh) [inline css/grid-3.css]: font-weight :normal is default value, it can be remove from here.

File: css/grid-3.css [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  color: var(--black);
+}
+
+/* Right Content */
+
+.cart-manage-container .right-content .primary-card-btn {
+  width: 100%;
+  color: var(--black);
+  background-color: var(--white);
+  border: 2px solid black;
```
Comment (sumitk-singh) [inline css/grid-3.css]: use color variable

**Review by sumitk-singh — CHANGES_REQUESTED** [review]
rest of the code looks clean, good to merger on the changes.

**Review by sonishreyas — APPROVED** [review]
Looks good to merge

**Pull request merged**